### PR TITLE
Add icons for munkireport

### PR DESCRIPTION
**Pull request by bochoven**

### CODE REVIEW

File: app/views/partials/head.php [hunk 1]
```diff
@@ -20,16 +19,14 @@
...
-  <link rel="apple-touch-icon" sizes="180x180" href="/assets/images/favicons/apple-touch-icon.png">
-  <link rel="icon" type="image/png" href="/assets/images/favicons/favicon-32x32.png" sizes="32x32">
-  <link rel="icon" type="image/png" href="/assets/images/favicons/favicon-16x16.png" sizes="16x16">
-  <link rel="manifest" href="/assets/images/favicons/manifest.json">
-  <link rel="mask-icon" href="/assets/images/favicons/safari-pinned-tab.svg" color="#5d5858">
-  <link rel="shortcut icon" href="/assets/images/favicons/favicon.ico">
-  <meta name="msapplication-config" content="/assets/images/favicons/browserconfig.xml">
-  <meta name="theme-color" content="#5d5858">
->>>>>>> cc42c704466aa8184a0a81a392aca1145c8d7db2
+  	<link rel="apple-touch-icon" sizes="180x180" href="/assets/images/favicons/apple-touch-icon.png">
```
Comment (bochoven) [inline app/views/partials/head.php]: Please prepend all paths with ```<?php echo conf('subdirectory'); ?>``` (see stylesheet directives above)

**Comment by WardsParadox** [comment]
Changed over the links to relative references using example, cleaned up a bad git merge which resulted with duplicated references. Should be good to merge now 👍

**Comment by bochoven** [comment]
The safari-pinned-tab.svg does not seem to work for me, the pinned tab is blank on my machine. (Safari 10.0 on Sierra GM)

**Comment by WardsParadox** [comment]
Hmm. That functionality of the generator seems to be broken as it won't let me upload a file for it. Let me do some fiddling with it some more.

**Comment by WardsParadox** [comment]
Seems to be working for me. I'm on safari version, same os x version, running a fresh pull as a test. Took awhile to generate but it shows up for me.

**Comment by bochoven** [comment]
thx!

**Pull request merged**